src/components/expansion-panel/themes/light/expansion-panel.base.scss

```diff
@@ -54,11 +54,11 @@ $padding: calc(var(--igc-spacing-large) * 2) calc(var(--igc-spacing-large) * 3);
     padding: $padding;
 }
 
-[part~='indicator'] {
+slot[name='indicator'] {
```

Comment:
It seems that custom styles targeted to the exposed parts do not have priority over the default styles, which target the slots.
In the styling sample, the following rule is expected to modify the indicator color:
`
igc-expansion-panel::part(indicator) {
    color: #ffd351;
}
`

However, with the current changes, this is the result:

![exp-panel-styles](https://user-images.githubusercontent.com/35433937/170471850-3cddff02-c4b1-4b67-9298-074cdcaaafec.png)
Same for the other exposed parts and slots (title, subtitle).

Reply:
@ddaribo should be fine now.

Reply:
The indicator is fine now.
However, setting the title and subtitle colors via the corresponding parts still does not seem to work, I guess because of the change icluding the ::slotted pseudo-element.